PackageInfo.g

```diff
@@ -104,7 +104,11 @@ ArchiveFormats := ".tar.gz",
 ##
 Status := "dev",
 
-AbstractHTML   :=  "",
+AbstractHTML   :=  "The <span class=\"smallgrp\">SmallGrp</span> package \
+provides the library of groups of certain "small" orders. The groups are \
```

Comment:
That's invalid syntax (you are using an unquoted `"` inside a string), thus merging this PR broke this package.